pkg/notes/options/options.go

```diff
@@ -230,13 +230,29 @@ func (o *Options) ValidateAndFinish() (err error) {
 
 	// Validate the output format and template
 	logrus.Infof("Using output format: %s", o.Format)
-	if o.Format == FormatMarkdown && !strings.HasPrefix(o.GoTemplate, GoTemplatePrefix) {
-		return errors.Errorf("go template has to be prefixed with %q", GoTemplatePrefix)
+	if o.Format == FormatMarkdown && o.GoTemplate != GoTemplateDefault {
```

Comment:
Looks like the linter failed because of this block. Can we factor it out so that the complexity drops again?